html/network.html

```diff
@@ -72,6 +103,71 @@ <h5 class="card-title"><strong>Output</strong></h5>
 
 
         <script>
+
+          function checkInterface(interfaceName, modal) {
+
+            // Clear output text
+            modal.find('h4#statusIndicator').html("<h4></h4>")
+            modal.find('p#outputText').prop("textContent", "")
+
+            // Disable test button while waiting for response
+            var button = document.getElementById("interface-" + interfaceName + "-test");
+            button.disabled = true;
+            // Also make it so the modal can't be closed while the test is running.
+            modal.find('button#closeButton').prop("disabled", true)
+            modal.find('button#crossCloseButton').prop("disabled", true)
+
+            // Show spinner
+            modal.find('div#testingSpinner').prop("hidden", false)
+            // document.getElementById("testingSpinner").hidden = false
+
+            fetch(
+              "/interface-status/" + interfaceName,
+              {method: "GET"}
+            ).then(function(response) {
+              return response.json();
+            }).then(function(json) {
+
+              // Hide spinner
+              // document.getElementById("testingSpinner").hidden = true
+              modal.find('div#testingSpinner').prop("hidden", true)
+
+              // Show output text.
+              modal.find('p#outputText').prop("textContent", json["result"])
+
+              // Show a symbol indicating whether the test was successfull or not.
+              var state = json["status"];
+              if (state === "down") {
+                // Cross mark
+                modal.find('h4#statusIndicator').html("<h4>&#x2717;</h4>")
+                // modal.find('.modal-body').html(json["result"] + "<h4 class=\"mt-3\">&#x2717;</h4>")
+              } else if (state === "up") {
+                // Check mark
+                modal.find('h4#statusIndicator').html("<h4>&#x2714;</h4>")
+                // modal.find('.modal-body').html(json["result"] + "<h4 class=\"mt-3\">&#x2714;</h4>")
+              } else {
+                // Question mark
+                modal.find('h4#statusIndicator').html("<h4>&#x003F;</h4>")
+                // modal.find('.modal-body').html(json["result"] + "<h4 class=\"mt-3\">&#x003F;</h4>")
```

Comment:
delete all these commented out bits

Reply:
Ok, doing this now.  Got the cards to be full screen up to 600px.  Working on the output padding...